src/app/components/server-compare-charts/server-compare-charts.component.scss

```
@@ -190,3 +194,42 @@
+.compare-table-header-baseline-marker {
+  position: absolute;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  height: 0.3rem;
+  background-color: #34d399;
+}
+
+.compare-table-metric-cell {
+  @apply ml-auto flex flex-col items-end gap-1;
+  width: fit-content;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)